mp-istio-sample/serviceA/src/main/webapp/WEB-INF/ibm-web-ext.xml

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE xml>
+<web-ext
```

Comment:
This should be in a liberty resource directory that's copied into the right place in /target during the build

Reply:
Thanks for the comments Ken. You can probably tell I'm feeling my way around this atm.